doc/api/net.md

```diff
@@ -224,8 +224,16 @@ added: v0.1.90
 -->
 
 Begin accepting connections on the specified `port` and `hostname`. If the
-`hostname` is omitted, the server will accept connections on any IPv6 address
-(`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+`hostname` is omitted, the server will accept connections on the [unspecified
+IPv6 address](https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address)
```

Comment:
Please move this link to the end also with the `[unspecified IPv6 address][]` inline